2025-01-20 20:58:46,641 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:58:47,221 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 20:58:48,104 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 20:58:48,106 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:58:48,113 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:58:48,123 INFO: 127.0.0.1 - - [20/Jan/2025 20:58:48] "GET /socket.io/?EIO=4&transport=polling&t=PI5IzTZ HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:58:48,141 INFO: 127.0.0.1 - - [20/Jan/2025 20:58:48] "POST /socket.io/?EIO=4&transport=polling&t=PI5Izb1&sid=86_n9t72Z1cQUYZ3AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:58:48,148 INFO: 127.0.0.1 - - [20/Jan/2025 20:58:48] "GET /socket.io/?EIO=4&transport=polling&t=PI5Izb6&sid=86_n9t72Z1cQUYZ3AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:18,484 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:18,732 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:19,217 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 20:59:19,881 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 20:59:19,882 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:19,890 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:19,943 INFO: 127.0.0.1 - - [20/Jan/2025 20:59:19] "GET /socket.io/?EIO=4&transport=polling&t=PI5J5Lu HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:19,956 INFO: 127.0.0.1 - - [20/Jan/2025 20:59:19] "POST /socket.io/?EIO=4&transport=polling&t=PI5J5MC&sid=7THG-frRvf41Xqf2AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:19,959 INFO: 127.0.0.1 - - [20/Jan/2025 20:59:19] "GET /socket.io/?EIO=4&transport=polling&t=PI5J5MF&sid=7THG-frRvf41Xqf2AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:59:19,969 INFO: 127.0.0.1 - - [20/Jan/2025 20:59:19] "GET /socket.io/?EIO=4&transport=polling&t=PI5J5MU&sid=7THG-frRvf41Xqf2AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:03,604 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:03,827 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:04,381 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 21:00:05,187 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 21:00:05,188 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:05,195 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:05,208 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:05] "GET /socket.io/?EIO=4&transport=polling&t=PI5JGJZ HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:05,222 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:05] "POST /socket.io/?EIO=4&transport=polling&t=PI5JGPT&sid=9Rc-xMLxgmyyQVnwAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:05,230 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:05] "GET /socket.io/?EIO=4&transport=polling&t=PI5JGPX&sid=9Rc-xMLxgmyyQVnwAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:18,527 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:18,744 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:19,178 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 21:00:19,839 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 21:00:19,840 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:19,848 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:20,019 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:20] "GET /socket.io/?EIO=4&transport=polling&t=PI5JK0c HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:20,034 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:20] "POST /socket.io/?EIO=4&transport=polling&t=PI5JK0u&sid=ieAxBM4wToC23JaUAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:20,039 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:20] "GET /socket.io/?EIO=4&transport=polling&t=PI5JK0-&sid=ieAxBM4wToC23JaUAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:00:20,050 INFO: 127.0.0.1 - - [20/Jan/2025 21:00:20] "GET /socket.io/?EIO=4&transport=polling&t=PI5JK1E&sid=ieAxBM4wToC23JaUAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:21,580 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:21,856 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:22,402 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 21:01:23,223 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 21:01:23,224 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:23,233 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:23,241 INFO: 127.0.0.1 - - [20/Jan/2025 21:01:23] "GET /socket.io/?EIO=4&transport=polling&t=PI5JZQA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:23,252 INFO: 127.0.0.1 - - [20/Jan/2025 21:01:23] "POST /socket.io/?EIO=4&transport=polling&t=PI5JZSj&sid=C5A0_UVVtdI7MxMQAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:23,252 INFO: 127.0.0.1 - - [20/Jan/2025 21:01:23] "GET /socket.io/?EIO=4&transport=polling&t=PI5JZSn&sid=C5A0_UVVtdI7MxMQAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:38,512 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:38,766 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:39,191 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 21:01:39,914 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 21:01:39,915 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:39,922 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:39,931 INFO: 127.0.0.1 - - [20/Jan/2025 21:01:39] "GET /socket.io/?EIO=4&transport=polling&t=PI5JdU- HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:39,948 INFO: 127.0.0.1 - - [20/Jan/2025 21:01:39] "POST /socket.io/?EIO=4&transport=polling&t=PI5JdXY&sid=CPQ_notBIDnFxXvmAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:01:39,955 INFO: 127.0.0.1 - - [20/Jan/2025 21:01:39] "GET /socket.io/?EIO=4&transport=polling&t=PI5JdXd&sid=CPQ_notBIDnFxXvmAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:02:23,505 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 21:02:23,722 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
